snapshots/overlay/overlay.go

```diff
@@ -518,6 +543,14 @@ func (o *snapshotter) mounts(s storage.Snapshot) []mount.Mount {
 		options = append(options, "userxattr")
 	}
 
+	if o.metacopyCtrl {
+		if o.metacopyOn {
+			options = append(options, "metacopy=on")
+		} else {
+			options = append(options, "metacopy=off")
```

Comment:
But with this change, then, it has the bug that I mentioned here: https://github.com/containerd/containerd/pull/7141#discussion_r929781602

If metacopy is set to false in the config but the kernel doesn't support it, we will append the `metacopy=off` option. I'm not sure that is incorrect (the user asked for it), but the `true` value is only used when it is also supported.

Reply:
Yes, sorry, fixed.